diff --git a/ios/bottom-tabs/host/RNSBottomTabsHostComponentView.mm b/ios/bottom-tabs/host/RNSBottomTabsHostComponentView.mm
index fc6257e36e7bff3e07c070201976f552b7defe07..dd174c474dfd34319822d028755ee8ac30747b81 100644
--- a/ios/bottom-tabs/host/RNSBottomTabsHostComponentView.mm
+++ b/ios/bottom-tabs/host/RNSBottomTabsHostComponentView.mm
@@ -114,7 +114,7 @@ - (void)resetProps
 #endif
   _tabBarTintColor = nil;
 #if !TARGET_OS_TV
-  _nativeContainerBackgroundColor = [UIColor systemBackgroundColor];
+  _nativeContainerBackgroundColor = [UIColor blackColor];
 #else // !TARGET_OS_TV
   _nativeContainerBackgroundColor = nil;
 #endif // !TARGET_OS_TV
@@ -319,7 +319,7 @@ - (void)updateProps:(const facebook::react::Props::Shared &)props
     _nativeContainerBackgroundColor = RCTUIColorFromSharedColor(newComponentProps.nativeContainerBackgroundColor);
 #if !TARGET_OS_TV
     if (_nativeContainerBackgroundColor == nil) {
-      _nativeContainerBackgroundColor = [UIColor systemBackgroundColor];
+      _nativeContainerBackgroundColor = [UIColor blackColor];
     }
 #endif // !TARGET_OS_TV
 
@@ -507,7 +507,7 @@ - (void)setNativeContainerBackgroundColor:(UIColor *_Nullable)nativeContainerBac
   _nativeContainerBackgroundColor = nativeContainerBackgroundColor;
 #if !TARGET_OS_TV
   if (_nativeContainerBackgroundColor == nil) {
-    _nativeContainerBackgroundColor = [UIColor systemBackgroundColor];
+    _nativeContainerBackgroundColor = [UIColor blackColor];
   }
 #endif // !TARGET_OS_TV
 
